applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -44,12 +46,45 @@ static struct net_buf_pool *iso_tx_pools[] = { LISTIFY(CONFIG_BT_ASCS_ASE_SRC_CO
 /* clang-format on */
 #endif /* CONFIG_STREAM_BIDIRECTIONAL */
 
+enum csis_set_rank { CSIS_HL_RANK = 1, CSIS_HR_RANK = 2, CSIS_SET_SIZE = 2 };
+
+static struct bt_csis *csis;
 static struct bt_le_ext_adv *adv_ext;
 /* Advertising data for peer connection */
+static uint8_t csis_rsi[BT_CSIS_RSI_SIZE];
+
 static const struct bt_data ad_peer[] = {
 	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
 	BT_DATA_BYTES(BT_DATA_UUID16_ALL, BT_UUID_16_ENCODE(BT_UUID_ASCS_VAL)),
-	BT_DATA_BYTES(BT_DATA_UUID16_ALL, BT_UUID_16_ENCODE(BT_UUID_PACS_VAL))
+	BT_DATA_BYTES(BT_DATA_UUID16_ALL, BT_UUID_16_ENCODE(BT_UUID_PACS_VAL)),
+	BT_CSIS_DATA_RSI(csis_rsi)
+};
+
+static void csis_lock_changed_cb(struct bt_conn *conn, struct bt_csis *csis, bool locked)
+{
+	LOG_DBG("Client %p %s the lock", (void *)conn, locked ? "locked" : "released");
+}
+
+static uint8_t sirk_read_req_cb(struct bt_conn *conn, struct bt_csis *csis)
+{
+	return BT_CSIS_READ_SIRK_REQ_RSP_ACCEPT_ENC;
+}
+
+static struct bt_csis_cb csis_callbacks = {
+	.lock_changed = csis_lock_changed_cb,
+	.sirk_read_req = sirk_read_req_cb,
+};
+struct bt_csis_register_param csis_param = {
+	.set_size = CSIS_SET_SIZE,
+	.lockable = true,
+#if !CONFIG_BT_CSIS_TEST_SAMPLE_DATA
```

Comment:
To me it seems like we should throw a #warning (or something similar) in both cases, since set_sirk also must be changed before production if CONFIG_BT_CSIS_TEST_SAMPLE_DATA=n

I guess CONFIG_BT_CSIS_TEST_SAMPLE_DATA is default 'n'?

Reply:
Will make a #print for indication in ordinary case (not using sample security data).
Warning for ordinary case might cause the test system screaming.

Reply:
We could do a LOG_WRN(), but that does not feel optimal

Reply:
If you have a look at the BT debug keys, they use a method which should be CI compatible.

Reply:
That will be need to be done in Zephyr side
https://github.com/zephyrproject-rtos/zephyr/blob/c0b87fd0e3011eaff12b4c32657e46c55383720a/subsys/bluetooth/host/CMakeLists.txt#L91
Should we check this config from our CMakelist or we improve from Zephyr side?

Reply:
To me that sounds like a smart thing to do